include/n64/PR/gu.h

```diff
@@ -191,7 +191,7 @@ extern float sinf(float angle);
 extern float cosf(float angle);
 extern signed short sins (unsigned short angle);
 extern signed short coss (unsigned short angle);
-extern float sqrtf(float value);
+#define sqrtf(v) __builtin_sqrtf(v)
```

Comment:
Can duplicate sqrtf defines throw any warnings or does it not matter if they define to the same thing?

Reply:
#define just replaces any existing defines without error

Reply:
Though it would cause an error if `sqrtf` were defined as a _function_ (not a macro) elsewhere. But that would already be the case with the current code

Reply:
Not without unsuppressible warnings (speaking from experience)

Reply:
You can guarantee no warnings if you undef it first, though it's not elegant if you use it everywhere

Reply:
Oh wait, yeah, you would want #ifndef first